src/helpers/url-helper.php

```diff
@@ -120,7 +120,7 @@ public function get_url_path( $url ) {
 	public function get_extension_from_url( $url ) {
 		$path = $this->get_url_path( $url );
 
-		if ( $path === '' ) {
+		if ( $path === '' || $path === null ) {
```

Comment:
In this case, this isn't actually needed as the `get_url_path()` method above does a `(string)` cast on the return value of the call to `wp_parse_url()`.